### DISCUSSION TITLE
Thoughts on the Android SDK

### DISCUSSION THREAD

**Discussion by xfqwdsj** [ScoopInstaller/Scoop#5355] [Ideas]
At present, the Android SDK in the official buckets is very confusing, I have implemented a better SDK management solution in [my own bucket](https://github.com/xfqwdsj/BucketDev) and am ready to request to merge them into the official buckets at any time.

There is an `android-sdk-base` manifest in the bucket, which is the base directory of the entire android sdk (can Scoop add modularization support? It must require `url` to work properly). Other components are installed in their own directories, but they create links within the `base` package. The advantage of managing components in this way is that you don't need to manually manage environment variables, and you can also use Android Studio or sdkmanager to manage SDK packages.

**Comment by HUMORCE** [comment]
`cmdline-tools` with setup scripts?

**Reply by xfqwdsj** [reply]
Yes, what I mainly want to solve is the problem of environment variables.  Installing this way eliminates the need to manually set environment variables and remove them when components are removed at a later date.  (Am I lazy enough 😂) If there is a better way, please help to point it out, thank you!
